notebooks/267-distil-whisper-asr/267-distil-whisper-asr.ipynb

```diff
@@ -15,7 +15,7 @@
     "\n",
```

Comment:
<p>Let's compare the transcription results for original <strong>and </strong>quantized models.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Fixed